tools/cityscapes/convert_cityscapes_to_coco.py

```diff
@@ -136,13 +150,14 @@ def convert_cityscapes_instance_only(
         'motorcycle',
         'bicycle',
     ]
-
+    
     for data_set, ann_dir in zip(sets, ann_dirs):
         print('Starting %s' % data_set)
         ann_dict = {}
         images = []
         annotations = []
         ann_dir = os.path.join(data_dir, ann_dir)
+        print('ann_dir: %15s'%ann_dir)
```

Comment:
nit: do we need this print?

Reply:
I found it useful to keep track of this, but not necessary.

How about adding tqdm progressbar? :)

Reply:
Why not, but it would add an extra dependency :-)
Maybe a print from time to time would be enough